matlab-src/whole_body_model_functions/wbm_getWorldFrameFromFixedLink.m

```diff
@@ -0,0 +1,33 @@
+function [ world_R_base,world_p_base ] = wbm_getWorldFrameFromFixedLink( varargin )
+%WBM_GETWORLDFRAMEFROMFIXEDLINK returns the floating base position for a
+%given reference link name and joint configuration
```

Comment:
The doc is a bit vague, it is not clear that "reference link" is the one that is coincident with world if the returned state is set as the world_R_base transform.